### ISSUE TITLE
line 2 column 54 - Error: <goal-name> is not recognized! dependency check maven plugin

### ISSUE THREAD

**Issue by DoumaRahma** [dependency-check/DependencyCheck#5080]
**Describe the bug**
We upgrade to version 7.3.1 and 7.3.2 of dependency check maven, the execution is completed successfully but we get this message 
***line 2 column 54 - Error: < goal-name > is not recognized!
     owasp_db : "7.3.2"***

while executing this command:  ` ./mvnw help:describe -Dplugin=org.owasp:dependency-check-maven $MAVEN_ARGS "${MAVEN_ADDITIONAL_ARGS_ARRAY[@]}" | grep Version:`

We don't have this message with version 7.3.0 of plugin neither with dependency check gradle v 7.3.2 and 7.3.1.

We make some other tests by changing the owasp DB version to 7.3.0, we have the same behavior. So, We assume that this message is related to the plugin not to the BD.
We even changed the version of maven-help-plugin with the latest one , we get the same result.
We used the grep with another plugin and it works without problem. So, we can confirm that the message comes from dependency-check-maven plugin.

We checked the plugin changes between version 7.3.2 and 7.3.0 https://github.com/jeremylong/DependencyCheck/compare/v7.3.0...v7.3.2 and there are only some visibility changes

**Version of dependency-check used**
The problem occurs using version 7.3.2 and 7.3.1 of dependency check maven plugin

**Expected behavior**
We expect to get the dependency check version as output.

**Comment by jeremylong** [comment]
Likely due to https://github.com/jeremylong/DependencyCheck/pull/5034.

Although, when I run this I get:

```shell
$ mvn help:describe -Dplugin=org.owasp:dependency-check-maven | grep 'Version'            
line 2 column 54 - Error: <goal-name> is not recognized!
Version: 7.3.2
```

The warning is being written to standard error - so could be hidden by running:

```shell
$ mvn help:describe -Dplugin=org.owasp:dependency-check-maven 2>/dev/null | grep 'Version'            
Version: 7.3.2
```

Also - your call has a several variables(maven args) that might be affecting the output. For instance, if the variable included `-q` - you would only get the warning message:

```shell
$ mvn help:describe -Dplugin=org.owasp:dependency-check-maven -q                          
line 2 column 54 - Error: <goal-name> is not recognized!
```

**Comment by DoumaRahma** [comment]
Thank you @jeremylong for your quick response.
We are asking this question since we haven't see this warning in versions under 7.3.1 and we are wondering if there are some changes in goal-name or something. At the end, we aren't blocked since we get the wanted version

**Comment by chadlwilson** [comment]
I don't think this is an issue now, or at least it will be rather different. Please open a new issue if still a problem on current ODC maven plugin
